frontend/next.config.mjs

```diff
@@ -1,10 +1,21 @@
-// @ts-check
+/** @type {import('next').NextConfig} */
+
+import nextra from 'nextra';
 
-/**
- * @type {import('next').NextConfig}
- */
 const nextConfig = {
-  /* config options here */
+  images: {
+    domains: ['ap-south-1.graphassets.com'],
+  },
+  typescript: {
+    // !! WARN !!
+    // Dangerously allow production builds to successfully complete even if
+    // your project has type errors.
+    // !! WARN !!
+    ignoreBuildErrors: true,
+  },
```

Comment:
We can't have this, we need to remove this

Reply:
Oh I will remove it, it's because I replaced this config with one of my project's.